frontend/public/components/namespace.jsx

```diff
@@ -95,12 +96,20 @@ const ProjectRow = ({obj: project}) => {
   </ResourceRow>;
 };
 
-const ProjectEmptyMessageDetail = <p>
-  Create a project for your application. To learn more, visit the OpenShift <a href={openshiftHelpBase} target="_blank" rel="noopener noreferrer">documentation</a>.
-</p>;
-const ProjectEmptyMessage = () => <MsgBox title="Welcome to OpenShift" detail={ProjectEmptyMessageDetail} />;
-export const ProjectList = props => <List {...props} Header={ProjectHeader} Row={ProjectRow} EmptyMsg={ProjectEmptyMessage} />;
-export const ProjectsPage = props => <ListPage {...props} ListComponent={ProjectList} canCreate={true} createHandler={createProjectModal} />;
+const ProjectListComponent = props => {
+  const ProjectEmptyMessageDetail = <p>
+    {props.createProjectMessage || 'Create a project for your application.'} To learn more, visit the OpenShift <a href={openshiftHelpBase} target="_blank" rel="noopener noreferrer">documentation</a>.
+  </p>;
+  const ProjectEmptyMessage = () => <MsgBox title="Welcome to OpenShift" detail={ProjectEmptyMessageDetail} />;
+  return <List {...props} Header={ProjectHeader} Row={ProjectRow} EmptyMsg={ProjectEmptyMessage} />;
+};
+export const ProjectList = connect(createProjectMessageStateToProps)(ProjectListComponent);
+
+const ProjectsPageComponent = props => {
+  const canCreate = !flagPending(props.flags.CAN_CREATE_PROJECT) && props.flags.CAN_CREATE_PROJECT;
```

Comment:
This could be

```js
const canCreate = props.flags.CAN_CREATE_PROJECT;
```

I understand you probably want to abstract away the pending logic, though. I'm OK either way.